lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -374,3 +413,63 @@ def _discard_tunnel_flows(self, precedence:int, i_teid:int,
             flows.delete_flow(self._datapath, self.tbl_num, match,
                               priority=priority + 1)
 
+    @set_ev_cls(ofp_event.EventOFPPacketIn, MAIN_DISPATCHER)
+    def _packet_in_handler(self, ev):
+        msg = ev.msg
+        pkt = packet.Packet(msg.data)
+        pkt_ipv4 = pkt.get_protocol(ipv4.ipv4)
+
+        dst = pkt_ipv4.dst
+        # For sending notification to SMF using GRPC
+        self._send_message_interface(dst, msg.cookie)
+        # Add flow for paging with hard time.
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP, ipv4_dst=dst)
+
+        flows.add_drop_flow(self._datapath, self.tbl_num, match, [],
+                       priority=flows.PAGING_PRIORITY + 1,
+                       hard_timeout= self.config.paging_timeout)
+
+    def _install_paging_flow(self, ue_ip_addr:IPAddress, local_f_teid:int, ng_flag: bool = True):
+        ofproto = self._datapath.ofproto
+        parser = self._datapath.ofproto_parser
+        ip_match_out = get_ue_ip_match_args(ue_ip_addr, Direction.IN)
+        match = MagmaMatch(eth_type=get_eth_type(ue_ip_addr), **ip_match_out)
+
+        # Pass Controller ID value as a ACTION
+        if ng_flag:
+            actions = [parser.NXActionController(0, self.config.classifier_controller_id,
+                                                 ofproto.OFPR_ACTION_SET)]
+        else:
+            actions = [parser.NXActionController(0, 0, ofproto.OFPR_ACTION_SET)]
```

Comment:
lets do
```
        classifier_controller_id = 0
        if ng_flag:
            classifier_controller_id = self.config.classifier_controller_id
        
         actions = [parser.NXActionController(0, classifier_controller_id, ofproto.OFPR_ACTION_SET)]
```

Reply:
I'm assuming ng_flag can change, otherwise we should just define the value exactly in config

Reply:
Changes done. ng_flag value is True for 5G and False for 4G.